core/capabilities/registry.go

```diff
@@ -14,16 +14,40 @@ var (
 	ErrCapabilityAlreadyExists = errors.New("capability already exists")
 )
 
+type metadataRegistry interface {
+	GetLocalNode(ctx context.Context) (capabilities.Node, error)
+	ConfigForCapability(ctx context.Context, capabilityID string, donID uint32) (capabilities.CapabilityConfiguration, error)
```

Comment:
nit/ `CapabilityConfig` would be inline with using `LocalNode` above

Reply:
@krehermann I'm not sure I'm following, wdym by inline here? re you suggesting we'd put the CapabilityConfiguration inside the Node struct? That doesn't work since it won't account for remote capabilities.